README.md

```diff
@@ -1,2 +1,107 @@
-# virtual-pet
-my virtual pet
+<a name="readme-top"></a>
```

Comment:
Theres a fair bit to talk about here

- README files should not be written using HTML, they are supposed to be written in Markdown language (Heres a helpful cheatsheet [guide to markdown](https://www.markdownguide.org/cheat-sheet/)). Also useful is [markdown editor](https://jbt.github.io/markdown-editor/) so you can play with what the file will look like without having to do multiple commits to the repo to see the results
- All the comments should be removed
- The linkedin shield should be at the bottom if anywhere, so it is with your contact info
- Your installation instructions start on 2
- The project link at the bottom just redirects to this page, it should be removed

The code itself is good, just needs a bit of tweaking to level it up a bit. A large part of that is just formatting so an easy fix